### PR TITLE
Fastly CLI 3.1.0

### PR DESCRIPTION
**Pull request by Integralist**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Updates the https://github.com/fastly/cli autocomplete spec to the latest `3.1.0` release.

**What is the current behavior? (You can also link to an open issue here)**

The current version in this repo is `2.0.0`.

### CODE REVIEW

**Comment by kailan** [comment]
thanks for doing this @Integralist –– it looks like the pre-commit formatter didn't run correctly because the diff is *huge*.

If you run `npm run lint:fix` it will resolve this and the diff should be more like what we saw in the figly repo: https://github.com/kailan/figly/commit/3237f16ad7fd7095f85b78b669fbe22368a01c04

**Review by kailan — APPROVED** [review]
✨ perfect 👍

**Review by mschrage — APPROVED** [review]

File: src/fastly.ts [hunk 1]
```diff
@@ -11111,7 +11247,7 @@ const completionSpec: Fig.Spec = {
                 },
                 {
                   name: "--address",
-                  description: "The hostname or IPv4 address",
+                  description: "The hostname or IPv4 addres",
```
Comment (SeparateRecords) [inline src/fastly.ts:11250]: Looks like this should be fixed upstream, but for now could you edit manually?

**Review by SeparateRecords — APPROVED** [review]
Looks good! Most of this diff is adding empty newlines, which can also be removed. If you're a vim user, should be as easy as `:%s/,\n\n/,\r/g` (replace comma followed by two newlines with a command and newline)

File: src/fastly.ts (hunk 1 above)
Comment (Integralist) [reply src/fastly.ts]: Fixed.

**Comment by Integralist** [comment]
@SeparateRecords regarding your comment...

> Looks good! Most of this diff is adding empty lines, which can also be removed. If you're a vim user, should be as easy as :%s/,\n\n/,\r/g (replace comma followed by two newlines with a comma and newline).

I am a [vim user](https://link.springer.com/book/10.1007/978-1-4842-0250-0)! 😉

I've now replaced all the line breaks. I think the tool I used to generate the updated file (https://github.com/kailan/figly) might have added those by accident.

**Comment by SeparateRecords** [comment]
Awesome, appreciate the changes. LGTM & thanks for updating the spec!!

(Also, that's incredible - I had no idea that was your book, I think I saw it recommended years ago while I was still in the steep part of the Vim learning curve!)

**Comment by fedeci** [comment]
I think we can integrate oss projects like fastly directly, and use diff versioned specs for them!

**Pull request merged**